agentstack/frameworks/crewai.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+from pathlib import Path
+import ast
+from . import SUPPORTED_FRAMEWORKS, ValidationError
+
+
+ENTRYPOINT: Path = Path('src/crew.py')
+
+def validate_project(path: Optional[Path] = None) -> None:
+    """
+    Validate that a CrewAI project is ready to run.
+    Raises a frameworks.VaidationError if the project is not valid.
+    """
+    try:
+        if path is None: path = Path()
+        with open(path/ENTRYPOINT, 'r') as f:
+            tree = ast.parse(f.read())
+    except (FileNotFoundError, SyntaxError) as e:
+        raise ValidationError(f"Failed to parse {ENTRYPOINT}\n {e}")
+
+    # A valid project must have a class in the crew.py file decorated with `@CrewBase`
+    try:
+        class_node = _find_class_with_decorator(tree, 'CrewBase')[0]
+    except IndexError:
+        raise ValidationError(f"`@CrewBase` decorated class not found in {ENTRYPOINT}")
+
+    # The Crew class must have one or more methods decorated with `@agent`
+    if len(_find_decorated_method_in_class(class_node, 'task')) < 1:
+        raise ValidationError(f"`@task` decorated method not found in `{class_node.name}` class in {ENTRYPOINT}")
```

Comment:
it might be worth giving the error message a hint like, `Try creating a Task with: agentstack generate task <name>`